source/common/event/timer_impl.cc

```diff
@@ -18,25 +18,21 @@ TimerImpl::TimerImpl(Libevent::BasePtr& libevent, TimerCb cb) : cb_(cb) {
 
 void TimerImpl::disableTimer() { event_del(&raw_event_); }
 
+void TimerImpl::millisecondsToTimeval(timeval* tv, const std::chrono::milliseconds& d) {
```

Comment:
Tiny nit: Envoy tends to have the order be input parameters followed by output parameters, and to use mutable ref. So, can this be `TimerImpl::millisecondsToTimeval(const std::chrono::milliseconds& d, timeval& tv)`?

Reply:
Sure! This is good advice regarding the code conventions.